src/molecules/singleLineInput/component.tsx

```diff
@@ -41,7 +41,7 @@ const getAutocompleteType = (inputType: InputType): string | undefined => {
 };
 
 export interface ISingleLineInputProps extends IMoleculeProps<ISingleLineInputTheme> {
-  value: string | null;
+  value: string | number | null;
```

Comment:
i dont think this will ever be a number? even if the user enters a number the output would be a string wouldnt it?